src/containers/custom-procedures.jsx

```diff
@@ -109,6 +113,7 @@ class CustomProcedures extends React.Component {
                 onAddLabel={this.handleAddLabel}
                 onAddTextNumber={this.handleAddTextNumber}
                 onCancel={this.handleCancel}
+                onKeyPress={this.handleOk}
```

Comment:
This will submit the modal any time a key is pressed, definitely not what you want. I think this line should be `onKeyPress={this.handleKeyPress}`